### PR TITLE
Update Kusto schema

### PR DESCRIPTION
**Pull request by radennis**

### CODE REVIEW

**Comment by Tiano2017** [comment]
@radennis seems there are conflicts.

**Comment by radennis** [comment]
Thanks @Tiano2017 fixed. Could you please take a look now?

**Comment by radennis** [comment]
@Tiano2017 could you please take a look?

File: schemas/2015-04-08/Microsoft.DocumentDB.json [hunk 1]
```diff
@@ -277,6 +277,17 @@
             }
           ],
           "description": "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists."
+        },
+        "isZoneRedundant": {
```
Comment (TianoMS) [inline schemas/2015-04-08/Microsoft.DocumentDB.json]: >isZoneRedundant [](start = 9, length = 15)

why are these changes included?

File: tests/2019-05-01/Microsoft.NetApp.tests.json [hunk 2]
```diff
@@ -0,0 +1,506 @@
+{
+  "tests": [
```
Comment (TianoMS) [inline tests/2019-05-01/Microsoft.NetApp.tests.json]: >tests [](start = 3, length = 5)

is this a merge conflict?

Comment (radennis) [reply tests/2019-05-01/Microsoft.NetApp.tests.json]: Not sure why it does that, diff against master shows only kusto's schema as changed. Anyway openined another PR, please take a look: https://github.com/Azure/azure-resource-manager-schemas/pull/617

**Comment by radennis** [comment]
Not sure why it does that, diff against master shows only kusto's schema as changed. Anyway opened another PR, please take a look: #617
Closing this PR
